Documentation/README.md

```diff
@@ -6,16 +6,25 @@ The following document includes documentation for the build scripts. Over time,
 
 The scripts are supported on Windows, macOS and Linux. The scripts are based on PowerShell on Windows and Bash on macOS and Linux.
 
-### Build on Windows
+### Syncing
+
+This repo uses submodules so remember on fresh clones or when updating/switching branches to run
+
+```
+git submodule update --init
+```
+
+### Building
 
 ```console
-./build.ps1
+./build.{cmd|sh}
 ```
 
-### Build on Linux or macOS
+### Cleaning
 
 ```console
-./build.sh
+./clean.{cmd|sh}      # Cleans root binary directory and resets all submodules
```

Comment:
I'm not sure how concerned to be about protecting people from accidentally cleaning away their work by only reading the readme, but "resets" is kind of vague. "fully git cleans and hard resets" would be explicit.

For best (imo) user experience, we would have `clean.sh/cmd` by default only run `git clean -xdn`, which lists everything it *would* delete (and do something similar for the `reset --hard`), then prompt. Then we could add `clean.sh/cmd -f` to do it without prompting.

Reply:
I don't really like the two stage process. However if folks end up hitting trouble then we can reconsider.